alembic/versions/2022-11-03-13.10.24_1923519381ad_renamed_timeline_event_message_and_.py

```diff
@@ -0,0 +1,31 @@
+"""renamed timeline event message and added recipe last made timestamp
+
+Revision ID: 1923519381ad
+Revises: 2ea7a807915c
+Create Date: 2022-11-03 13:10:24.811134
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "1923519381ad"
+down_revision = "2ea7a807915c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    # we need to rename the timeline event message field due to a frontend plugin conflict
+    op.alter_column("recipe_timeline_events", "message", new_column_name="event_message")
```

Comment:
alter_column isn't supported in SQLite. If it's just a name conflict can we rewrite it using a pydantic alias?

Reply:
Oh fun. Yeah I can do that too!